### PR TITLE
Golang update 1.18

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

**Review by sameshai — APPROVED** [review]

File: pkg/ibmcsidriver/node.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ func (csiNS *CSINodeServer) NodeGetVolumeStats(ctx context.Context, req *csi.Nod
 	ctxLogger, requestID := utils.GetContextLogger(ctx, false)
 	ctxLogger.Info("CSINodeServer-NodeGetVolumeStats... ", zap.Reflect("Request", *req)) //nolint:staticcheck
 	metrics.UpdateDurationFromStart(ctxLogger, "NodeGetVolumeStats", time.Now())
-	if req == nil || req.VolumeId == "" {
+	if req == nil || req.VolumeId == "" { //nolint
```
Comment (arahamad) [inline pkg/ibmcsidriver/node.go:265]: This looks be logic error, it should be && right?

File: go.mod [hunk 2]
```diff
@@ -17,11 +16,70 @@ require (
...
 	google.golang.org/grpc v1.38.0
-	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 	k8s.io/kubernetes v1.21.0
 	k8s.io/mount-utils v0.21.0
 )
 
+require (
+	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.6.7 // indirect
+	github.com/IBM/ibmcloud-volume-vpc v1.0.0-beta15 // indirect
```
Comment (arahamad) [inline go.mod:26]: why we need this lib?

**Review by arahamad — CHANGES_REQUESTED** [review]
can you check comments

File: go.mod (hunk 2 above)
Comment (prankulmahajan) [reply go.mod:26]: This lib is used is csi-common which is used by file-csi-driver.
Going forward will remove this

**Review by arahamad — APPROVED** [review]
lgtm, we need to remove dependency for github.com/IBM/ibmcloud-volume-vpc

**Pull request merged**